Ryujinx.Horizon.Generators/CodeGenerator.cs

```diff
@@ -1,12 +1,14 @@
-﻿using System.Text;
+﻿using System;
+using System.Text;
 
 namespace Ryujinx.Horizon.Generators
 {
     class CodeGenerator
     {
-        private const string Indent = "    ";
+        private const int SingleIndentLength = 4;
+
         private readonly StringBuilder _sb;
-        private string _currentIndent;
+        private int _currentIndentCharsCount;
```

Comment:
```suggestion
        private int _currentIndentCount;
```
I think the name doesn't need to be this long to understand the purpose of this field.